fields/types/date/DateType.js

```diff
@@ -93,6 +93,8 @@ date.prototype.moment = function (item) {
  */
 date.prototype.parse = function (value, format) {
 	var m = this.isUTC ? moment.utc : moment;
+	// TODO Check should maybe be if (typeof value === 'string')
+	// use the parseFormatString. Ever relevant?
 	if (typeof value === 'number' || value instanceof Date) {
```

Comment:
This should maybe be

```JS
if (typeof value === 'string') {
  return m(value, format || this.parseFormatString);
} else {
  return m(value);
}
```

instead of the other way around – does that ever matter?